### DISCUSSION TITLE
Default Field Values per Language

### DISCUSSION THREAD

**Discussion by Xinnerpeace** [directus/directus#23423] [Feature Requests]
### Summary

The translation feature in Directus is incredibly useful, but I’ve noticed a limitation: there’s currently no way to set different default values for different languages. 

In the schema settings for a field, the default value can only be set to a single content, meaning the default value is the same regardless of the selected language. This doesn’t align with the typical use case where different languages might require different default values.

Suggestion: Add the ability to set default values per language in the schema settings of the Translation table fields. This would provide greater flexibility for multilingual projects and enhance Directus's usability in internationalization scenarios.

Thank you for considering this feature, and I appreciate the hard work of the development team!
![image](https://github.com/user-attachments/assets/1a1eec16-951f-4a63-8bc3-65400cf81dfd)

### Motivation

Add the ability to set default values per language in the schema settings of fields.

### Detailed Design

![image](https://github.com/user-attachments/assets/25983609-a4af-4f60-8015-b34970bcf7d2)

### Must Have:
- Per-Language Default Value Setting: Allow users to set different default values for each language in the Translation table schema.
- UI Update
- Backward Compatibility: Ensure the feature doesn’t break existing projects. If no language-specific default is set, fallback to the global default.
### Should Have:
- Batch Editing: Enable setting or updating default values for multiple languages simultaneously to streamline the process.
- Documentation Update
### Could Have:
- Language-Specific Fallback Logic: Allow customizable fallback logic if no default is set for a specific language (e.g., fallback to English).

### Alternatives

Workarounds: 
- Users can currently implement workarounds in user space, such as using Flows to set default values based on the language selected. While not as elegant or integrated, these solutions can achieve the same result without requiring changes to the core Directus platform.
- Customization via Extensions

**Comment by JoshTheDerf** [comment]
This will be challenging to implement, since (I assume) column defaults are stored in the database row definition, not Directus itself.

Adding translations would mean removing the default value, which would cause default values to no longer be set when inserting data into the database outside of Directus's API.

```sql
CREATE TABLE mytable {
 mycolumn VARCHAR(255) DEFAULT 'mydefaultvalue'
);
```

**Reply by rijkvanzanten** [reply]
That's exactly right
